app/src/interfaces/input-rich-text-html/input-rich-text-html.vue

```diff
@@ -299,10 +299,12 @@ export default defineComponent({
 							const params = new URLSearchParams(matched.search);
 							params.delete('amp;access_token');
 							params.delete('access_token');
-							return `${pre}${matched.origin}${matched.pathname}${params}${post}`;
+							return `${pre}${matched.origin}${matched.pathname}${
+								props.imageToken ? props.imageToken : ''
+							}${params}${post}`;
```

Comment:
Wouldn't this resolve into `https://localhost:8055/assets/abc-def-ghi/my-custom-image-token?query=ab` ? The imageToken is part of params, so I think it has to be

```
params.set('access_token', props.imageToken);

return `${pre}${matched.origin}${matched.pathname}${params}${post}`;